docker/base.dockerfile

```diff
@@ -22,13 +23,16 @@ RUN apt update && apt install -y \
   librtmp-dev \
   libx264-dev \
   ninja-build \
+  ocl-icd-opencl-dev \
```

Comment:
Should we put all of those in doc/BUILD.md as well?

Reply:
:+1: